homeassistant/util/yaml/loader.py

```diff
@@ -4,13 +4,22 @@
 from collections import OrderedDict
 from collections.abc import Iterator
 import fnmatch
+from io import StringIO
 import logging
 import os
 from pathlib import Path
 from typing import Any, TextIO, TypeVar, Union, overload
 
 import yaml
 
+try:
+    from yaml import CSafeLoader as FastestAvailableSafeLoader
+
+    HAS_C_LOADER = True
+except ImportError:
+    HAS_C_LOADER = False
+    from yaml import SafeLoader as FastestAvailableSafeLoader  # type: ignore[misc]
+
```

Comment:
This is adapted from pyyaml's recommended incantation here https://pyyaml.org/wiki/PyYAMLDocumentation